boot/mbed/mcuboot_imgtool.cmake

```diff
@@ -107,10 +107,14 @@ function(_mcuboot_generate_image TARGET IMAGE_TYPE IMAGE_BASENAME)
 
             ${TARGET_HEX_FILE} ${IMAGE_BASE_PATH}.hex
 
+        COMMAND ${CMAKE_COMMAND} -E echo "-- built: ${IMAGE_BASE_PATH}.hex"
+
         # Also generate bin file
         COMMAND
             ${objcopy} -I ihex -O binary ${IMAGE_BASE_PATH}.hex ${IMAGE_BASE_PATH}.bin
 
+        COMMAND ${CMAKE_COMMAND} -E echo "-- built: ${IMAGE_BASE_PATH}.bin"
```

Comment:
why do you need echos here?

Reply:
use the cmake `message()` function https://cmake.org/cmake/help/latest/command/message.html#general-messages with `STATUS`

Reply:
Ah, unfortunately message() can only print stuff at configure time.  I want this message to print at build time after the hex and bin files have been built.